backend/workflowAPI/prisma/schema.prisma

```diff
@@ -16,10 +16,13 @@ enum TaskType {
 }
 
 enum TaskAction {
-    GITHUB_PR_MERGE    // Action
-    GITHUB_ISSUE_CLOSE // Action & reaction
-    TIMER_DATE         // Action
-    TIMER_INTERVAL     // Action
+  GITHUB_PR_OPEN            // Action
+  GITHUB_ISSUE_OPEN         // Action
+  TIMER_DATE                // Action
+  TIMER_INTERVAL            // Action
+  GOOGLE_CALENDAR_NEW_EVENT // Reaction
+  GOOGLE_NEW_DOC            // Reaction
+  GOOGLE_NEW_SHEET          // Reaction
```

Comment:
Same as above.

Reply:
Fixed in https://github.com/Cleopha/ifttt-like/pull/42/commits/5bd497b96dbec7651fb3c60fdd9d87ffbf3a89de